e2e-tests/cypress/tests/integration/channels/enterprise/system_console/channel_moderation/helpers.ts

```diff
@@ -62,14 +65,14 @@ export const saveConfigForChannel = (channelName = false, clickConfirmationButto
 };
 
 // # Visits a channel as the member specified
-export const visitChannel = (user, channel, team) => {
-    cy.apiLogin(user);
+export const visitChannel = (user: User, channel: Channel, team: Team) => {
+    cy.apiLogin(user as UserProfile);
```

Comment:
Can we just have the function take in a `UserProfile` if we're going to be immediately casting it?

Reply:
I'll change it to UserProfile